create-node-api/packaging/postinst

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# add post-installation script, like:
+
+# psql -U rudder -h localhost -d rudder -f /opt/rudder/share/plugins/createnodeapi/Createnodeapi-schema.sql
```

Comment:
Should we keep this comment ? I think we need the file

Reply:
I prefer to keep it.